master/internal/config/authz_config.go

```diff
@@ -86,6 +90,26 @@ func (c AuthZConfig) IsRBACUIEnabled() bool {
 	return c.Type != BasicAuthZType
 }
 
+// IsRBACEnabled returns if the authz config type is using the RBAC implementation
+// and will attempt to use Fallback and Default types if necessary.
+func (c AuthZConfig) IsRBACEnabled() bool {
```

Comment:
suggestion: I'd put it next to isrbacuienabled and differentiate them

Reply:
@hamidzr what do you mean by next to it, like an additional comment? This is right below `IsRBACUIEnabled`

Reply:
my bad I only checked lines coming after this so I didn't realize it's right below it. all good